app-modules/form/src/Filament/Resources/FormResource/Pages/EditForm.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Assist\Form\Filament\Resources\FormResource\Pages;
+
+use Assist\Form\Models\Form;
+use Filament\Actions\Action;
+use Filament\Actions\DeleteAction;
+use Filament\Forms\Components\Builder;
+use Assist\Form\Filament\Blocks\Select;
+use Illuminate\Database\Eloquent\Model;
+use Filament\Forms\Form as FilamentForm;
+use Filament\Resources\Pages\EditRecord;
+use Assist\Form\Filament\Blocks\TextArea;
+use Assist\Form\Filament\Blocks\TextInput;
+use Assist\Form\Filament\Resources\FormResource;
+use Filament\Forms\Components\Textarea as FilamentTextarea;
+use Filament\Forms\Components\TextInput as FilamentTextInput;
+
+class EditForm extends EditRecord
+{
+    protected static string $resource = FormResource::class;
+
+    protected static ?string $navigationLabel = 'Edit';
+
+    public function form(FilamentForm $form): FilamentForm
+    {
+        return parent::form($form)
+            ->schema([
+                FilamentTextInput::make('name')
+                    ->required()
+                    ->string()
+                    ->maxLength(255)
+                    ->autocomplete(false)
+                    ->columnSpanFull(),
+                FilamentTextarea::make('description')
+                    ->string()
+                    ->columnSpanFull(),
+                Builder::make('content')
```

Comment:
Should this be called `form_fields` or something like that?

Reply:
`form_fields` feels a bit wordy to me, so I just went with fields, and I moved the builder into a section to differentiate it more.

![image](https://github.com/canyongbs/assist/assets/11588575/6567a86e-4483-4a63-9fbe-1d6577a95204)